.gitignore

```diff
@@ -150,4 +150,6 @@ dmypy.json
 .pytype/
 
 # Cython debug symbols
-cython_debug/
\ No newline at end of file
+cython_debug/
+buzz_el/rdf_kg_2_el_loader.py
```

Comment:
does it make sense to keep this here ?

Reply:
The cython debug I would say yes. It comes from a standard python project gitignore.

Reply:
Sorry, my comment was on line 154 `buzz_el/rdf_kg_2_el_loader.py`, this is for your local right ?

Reply:
Yes it wa my local file, but I removed it. Didn't I ?